.cirrus.yml

```diff
@@ -18,6 +18,8 @@ build: &BUILD
     - . $HOME/.cargo/env || true
     - $TOOL +$TOOLCHAIN -Vv
     - rustc +$TOOLCHAIN -Vv
+    - if [ -z "$TOOLCHAIN" ]; then rustup component add rustfmt; else rustup +$TOOLCHAIN component add rustfmt; fi
+    - $TOOL +$TOOLCHAIN fmt --all -- --check
```

Comment:
`cargo fmt` ignores cfg directives, so running it once suffices to provide coverage for every OS.  Running it on every OS is duplicative.  Instead, you should either create a new task just for this, or else put it into the "Rust Stable" task.

Reply:
I have created a new CI task. This way, if this is the only test failing, the person who submitted the PR will easily know that the PR failed due to bad formatting. I have tested the commit without applying the formatting changes and [everything seems fine](https://cirrus-ci.com/build/4684991404703744)

Reply:
I have also added the `rustfmt.toml` file specifying that he max width needs to be 80 characters.